pulpcore.te

```diff
@@ -9,58 +9,138 @@ type pulpcore_t;
 type pulpcore_exec_t;
 init_daemon_domain(pulpcore_t, pulpcore_exec_t)
 init_nnp_daemon_domain(pulpcore_t)
+permissive pulpcore_t;
+
+type pulpcore_server_t;
+type pulpcore_server_exec_t;
+init_daemon_domain(pulpcore_server_t, pulpcore_server_exec_t)
+init_nnp_daemon_domain(pulpcore_server_t)
+permissive pulpcore_server_t;
+
+type pulpcore_etc_t;
+files_config_file(pulpcore_etc_t)
 
 type pulpcore_var_lib_t;
 files_type(pulpcore_var_lib_t)
+type pulpcore_server_var_lib_t;
+files_type(pulpcore_server_var_lib_t)
 
 type pulpcore_var_run_t;
 files_pid_file(pulpcore_var_run_t)
+type pulpcore_server_var_run_t;
+files_pid_file(pulpcore_server_var_run_t)
```

Comment:
I'd like to remove the use of pid files completely so perhaps we should do that before merging this? (and also get rid of `files_pid_filetrans` below)

Reply:
If the service runtime content is at a different place, we can remove it. Just note with a generic `var_run_t` type for the pidfile the service will not have access to it.